pyperformance/data-files/benchmarks/bm_ctypes/pyproject.toml

```
@@ -0,0 +1,10 @@
+[project]
+name = "pyperformance_bm_ctypes"
+requires-python = ">=3.8"
+dependencies = ["pyperf", "setuptools"]
+urls = {repository = "https://github.com/python/pyperformance"}
+dynamic = ["version"]
+
+[tool.pyperformance]
+name = "ctypes"
+tags = "extension"
```

```suggestion
tags = "extension"

```
File missing trailing whitespace.